src/main/resources/data/po-lines/38434-1_pending_fomat-other.json

```diff
@@ -0,0 +1,99 @@
+{
```

Comment:
Please Double check... doesn't have physical or eresource...  Other orders should probably have physical

Reply:
38434-1 had neither a physical nor eresource listing in master

Reply:
OK, let's leave it as-is for now.  We can change it later if we need to